src/demo-app/stepper/stepper-demo.html

```diff
@@ -1,3 +1,49 @@
+<h2>Linear Vertical Stepper Demo</h2>
+<md-vertical-stepper>
+  <md-step>
+    <form [formGroup]="nameFormGroup" novalidate>
```

Comment:
Remove `novalidate`? It's added automatically by the forms module. Same in forms below.